engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -255,17 +255,21 @@ private void initRenderGraph() {
         BufferClearingNode reflectedRefractedClearingNode = new BufferClearingNode(reflectedRefractedBufferConfig, displayResolutionDependentFBOs, GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
         renderGraph.addNode(reflectedRefractedClearingNode, "reflectedRefractedClearingNode");
 
-        FBOConfig sceneOpaqueFboConfig = displayResolutionDependentFBOs.getFboConfig(BUFFER1);
+        FBOConfig scenePrimaryBufferConfig = displayResolutionDependentFBOs.getFboConfig(BUFFER1);
```

Comment:
Done to make it match the functions getPrimaryBuffer and getSecondaryBuffer. I expect that (and subsequently this) to change after @emanuele3d's comments.